ui/apps/profile/src/components/pages/profile-engagement/followers-page.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useMemo } from 'react';
+import FollowProfile from '../../follow-profile';
+import Followers from '@akashaorg/design-system-components/lib/components/ProfileEngagements/Engagement/Followers';
+import EngagementTab from './engagement-tab';
+import Box from '@akashaorg/design-system-core/lib/components/Box';
+import EntryError from '@akashaorg/design-system-components/lib/components/ProfileEngagements/Entry/EntryError';
+import { useTranslation } from 'react-i18next';
+import { RootComponentProps } from '@akashaorg/typings/ui';
+import { useParams } from 'react-router-dom';
+import {
+  useGetProfileByDidQuery,
+  useInfiniteGetFollowersListByDidQuery,
+} from '@akashaorg/ui-awf-hooks/lib/generated/hooks-new';
+import { getProfileImageVersionsWithMediaUrl, useGetLogin } from '@akashaorg/ui-awf-hooks';
+import { ProfileEngagementLoading } from '@akashaorg/design-system-components/lib/components/ProfileEngagements/placeholders/ProfileEngagementLoading';
+
+const FollowersPage: React.FC<RootComponentProps> = props => {
+  const { t } = useTranslation('app-profile');
+  const { profileId } = useParams<{ profileId: string }>();
+  const loginQuery = useGetLogin();
+  const navigateTo = props.plugins['@akashaorg/app-routing']?.routing?.navigateTo;
+  const profileDataReq = useGetProfileByDidQuery(
+    {
+      id: profileId,
+    },
+    {
+      select: response => response.node,
+      enabled: !!loginQuery.data?.id,
+    },
+  );
+  const followersReq = useInfiniteGetFollowersListByDidQuery(
+    'last',
+    {
+      id: profileId,
+      last: 10,
+    },
+    {
+      enabled: !!loginQuery.data?.id,
+    },
+  );
+  const followers = useMemo(
+    () =>
+      followersReq.data?.pages
+        ? followersReq.data.pages?.flatMap(page =>
+            'isViewer' in page.node
+              ? page.node?.profile?.followers?.edges?.map(edge => edge?.node) || []
+              : [],
+          )
+        : [],
+    [followersReq.data],
+  );
+  const isViewer =
+    profileDataReq.data && 'isViewer' in profileDataReq.data ? profileDataReq.data.isViewer : null;
```

Comment:
I would suggest using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn for checking if a property exists

Reply:
`in` iterates over all the prototype chain